src/main/java/no/sikt/nva/BrageProcessor.java

```diff
@@ -64,7 +66,8 @@ public BrageProcessor(String zipfile,
                           boolean enableOnlineValidation,
                           boolean shouldLookUpInChannelRegister,
                           boolean noHandleCheck,
-                          List<Embargo> embargoes) {
+                          List<Embargo> embargoes,
+                          List<Contributor> contributors) {
```

Comment:
Bruk hashset istedet. Opprettelse av hashet tar lengre tid enn for List, men oppslag er O(1).
Så navnet du skal slå opp på er key, og contributor er value i hashset'et.

Reply:
HashMap tenker du?